test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -295,6 +309,25 @@ func (h *holder) verifyObject(obj runtime.Object) error {
 	return nil
 }
 
+func (h *holder) verifyOptions(options runtime.Object) error {
+	if options == nil {
+		return fmt.Errorf("no options sent")
+	}
+	if options.GetObjectKind().GroupVersionKind() != h.expectOptionsGVK {
+		return fmt.Errorf("expected %#v, got %#v", h.expectOptionsGVK, options.GetObjectKind().GroupVersionKind())
+	}
+	if options.GetObjectKind().GroupVersionKind().Kind == "DeleteOptions" {
+		data := options.(*unstructured.Unstructured).UnstructuredContent()
+		if _, ok := data["gracePeriodSeconds"]; !ok {
```

Comment:
why these specific field validations?

Reply:
This was for my own sanity checking, but since we only test GVK here, I'll remove.